frontend/automation/cypress/e2e/wallet-permissions.cy.js

```diff
@@ -0,0 +1,60 @@
+const {
+  authenticate,
+  unlockWallet,
+  approveConnection
+} = require('../support/helpers')
+
+const testIds = {
+  SELECTION_MODAL: 'wallet-selection-modal',
+  REJECT_CONNECTION_BUTTON: 'wallet-connection-reject',
+  APPROVE_SELECTION_BUTTON: 'wallet-connection-approve'
+}
+
+describe('wallet permissions', () => {
+  const MOCK_HOSTNAME = 'https://best-blockchain.app'
+  let walletName
+  let passphrase
+  let clientToken
+
+  before(() => {
+    cy.clean()
+    cy.backend().then(handler => {
+      cy.setVegaHome(handler)
+      cy.restoreNetwork(handler)
+      cy.restoreWallet(handler)
+    })
+  })
+
+  beforeEach(() => {
+    passphrase = Cypress.env('testWalletPassphrase')
+    walletName = Cypress.env('testWalletName')
+
+    cy.waitForHome()
+    // approveConnection(MOCK_HOSTNAME, walletName, passphrase)
+  })
+
+  it('handles approval', () => {
+    // cy.sendPermissionsRequest(MOCK_HOSTNAME, {
```

Comment:
this will need to be fixed, but i need some QA person for it, spent too much time on it and couldn't figure it out unfortunately.